Tests/ProtobufferCodingTests/ProtobufEncoderTests.swift

```diff
@@ -10,69 +10,124 @@ import XCTest
 @testable import ProtobufferCoding
 
 class ProtobufEncoderTests: XCTestCase {
```

Comment:
Shall we commit to always write Protobuffer instead of abbreviations?

Reply:
Good point! We definitely should stick to one version.
Let's go for writing it out, as you suggested 👍